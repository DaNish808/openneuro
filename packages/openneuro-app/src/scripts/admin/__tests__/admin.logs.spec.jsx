import React from 'react'
import { shallow } from 'enzyme'
import Logs from '../admin.logs'

describe('admin/Logs', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<Logs />)
    expect(wrapper).toMatchSnapshot()
  })
})
