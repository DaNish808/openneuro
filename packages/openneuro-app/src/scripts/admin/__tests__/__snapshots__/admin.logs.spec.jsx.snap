// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin/Logs renders successfully 1`] = `
<div
  className="dashboard-dataset-teasers fade-in admin-users clearfix"
>
  <div
    className="header-wrap clearfix"
  >
    <div
      className="col-sm-9"
    >
      <h2>
        Events Log
      </h2>
    </div>
    <div
      className="col-sm-3"
    >
      <Input
        className="pull-right"
        onChange={[Function]}
        placeholder="Search Type or Email"
      />
    </div>
  </div>
  <div>
    <div
      className="col-xs-12 users-panel-wrap"
    >
      <div
        className="fade-in user-panel-header clearfix"
      >
        <div
          className="col-xs-3 user-col"
        >
          <label>
            Event Type
          </label>
        </div>
        <div
          className="col-xs-3 user-col"
        >
          <label>
            User
          </label>
        </div>
        <div
          className="col-xs-3 user-col"
        >
          <label>
            Date
          </label>
        </div>
        <div
          className="col-xs-3 user-col"
        >
          <label>
            Link
          </label>
        </div>
      </div>
      <p
        className="no-datasets"
      >
        There are no event logs.
      </p>
    </div>
  </div>
  <div
    className="pager-wrapper"
  >
    <Paginator
      activePageRangeDisplayed={1}
      breakLabel="..."
      containerClass="pagination"
      nextLabel="»"
      onPageSelect={[Function]}
      page={0}
      pageRangeDisplayed={5}
      pagesTotal={0}
      prevLabel="«"
    />
  </div>
</div>
`;
